### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by 3y3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     },
     "./runtime": {
       "types": "./runtime/index.d.ts",
-      "style": "./runtime/styles.css",
+      "style": "./runtime/index.css",
```
Comment (vsesh) [inline package.json:14]: Does it break the backward compatibility?

File: src/runtime/index.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import {GLOBAL_SYMBOL} from '../common';
 import {TabsController} from './TabsController';
+import './scss/tabs.scss';
```
Comment (vsesh) [inline src/runtime/index.ts:3]: Services will have to add `unsafe-inline` in the CSP config. And it breaks the backward compatibility too.

Comment (vsesh) [reply src/runtime/index.ts:3]: I think it is bad to ask users to add unsafe CSP rules. We should somehow proxy nonce in the current case.

File: package-lock.json [hunk 3]
```diff
@@ -1,78 +1,95 @@
 {
   "name": "@diplodoc/tabs-extension",
-  "version": "2.0.8",
-  "lockfileVersion": 3,
+  "version": "2.0.9",
+  "lockfileVersion": 1,
```
Comment (vsesh) [inline package-lock.json:4]: You should use node version from project's `nvm`

Comment (3y3) [reply package-lock.json:4]: There is no `.nvmrc`.
This is also a problem of some last fixes, that plugin uses node version incompatible with core tech.

File: src/runtime/index.ts (hunk 2 above)
Comment (3y3) [reply src/runtime/index.ts:3]: There in no problem with csp. This is not inline css.

File: package.json (hunk 1 above)
Comment (3y3) [reply package.json:14]: Yep. This will change import on transformer side.
Do you know other broken points?

Comment (vsesh) [reply package.json:14]: No. I just wanted to highlight it.

**Review by vsesh — APPROVED** [review]

File: package-lock.json (hunk 3 above)
Comment (vsesh) [reply package-lock.json:4]: Got it. I regenerated `package-lock` for nodejs=14

Comment (vsesh) [reply package-lock.json:4]: And I added `.nvnrc`

**Review by moki — APPROVED** [review]

**Pull request merged**